WS2012R2/lisa/setupscripts/STOR_VSS_Utils.ps1

```diff
@@ -247,7 +257,7 @@ function checkResults([string] $vmName, [string] $hvServer)
 
 	# Now Check the boot logs in VM to verify if there is no Recovering journals in it .
 	$sts=CheckRecoveringJ
-    if ($sts[-1])
+    if (-not $sts[-1])
```

Comment:
I've run the tests on two different VM's and this would fail or not depending on the Linux Flavor that was being used. This might be related to where the Recovering Journal is stored on each VM.